packages/horizontal-slider-react/src/HorizontalSlider.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useEffect, useMemo, useRef } from 'react';
+
+let lastHorizontalSliderId = 0;
+
+function generateHorizontalSliderId() {
+  return `uic-horizontal-slider-${lastHorizontalSliderId++}`;
+}
+
+function cx(...classNames: Array<string | undefined>) {
+  return classNames.filter(Boolean).join(' ');
+}
+
+type RecordWithObjectID<TObject = {}> = TObject & {
+  objectID: string;
+};
+
+type HorizontalSliderClassnames = {
+  item: string;
+  list: string;
+  navigation: string;
+  navigationNext: string;
+  navigationPrevious: string;
+  root: string;
+};
+
+type HorizontalSliderTranslations = {
+  sliderLabel: string;
+  previousButtonLabel: string;
+  previousButtonTitle: string;
+  nextButtonLabel: string;
+  nextButtonTitle: string;
+};
```

Comment:
Why don't we export those types?

If we do so, it should be encapsulated with `Partial<{...}>`, otherwise, the user won't have the "real" type and will have to do it on his side

Reply:
See 17462ff5628938a3592e4d77da0bab7fff5b6497.